README.md

```diff
@@ -122,8 +134,8 @@ the content of the bundle available in the cluster.
 ### CustomResourceDefinition (CRD) Validator
 
 RukPak comes with a webhook for validating the upgrade of CRDs from `Bundle`s. If a CRD does potentially destructive
-actions to the cluster, it will not allow it go be applied. In the context of RukPak, this will result in a failed 
-`BundleInstance` resolution. 
+actions to the cluster, it will not allow it go be applied. In the context of RukPak, this will result in a failed
```

Comment:
May as well fix it while we're here 🤷🏻 
```suggestion
actions to the cluster, it will not allow it to be applied. In the context of RukPak, this will result in a failed
```